### PR TITLE
Profiling tweak and docs

### PR DESCRIPTION
**Pull request by vext01**
See commits.

### CODE REVIEW

File: docs/src/dev/profiling.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Profiling
+
+This section describes how best to profile yk (and its consumers).
+
+Note that `yk-config --cflags` includes `-Wl,--no-rosegment`, which is [gives
```
Comment (ltratt) [inline docs/src/dev/profiling.md:5]: s/is//

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**